tests/Project.py

```diff
@@ -187,3 +187,29 @@ def testCreateCardFromIssue(self):
         issue = self.repo.create_issue(title="Issue created by PyGithub")
         card2 = column.create_card(content_id=issue.id, content_type="Issue")
         self.assertEqual(card2.id, 16039106)
+
+    def testEditCardWithoutParameters(self):
+        project = self.repo.create_project(
```

Comment:
I was checking into this yesterday, and I think you would be better placed grabbing the project by id directly and making use of that replay data. For example, as seen on line 149 in this file.

Reply:
Thanks for the suggestion! As you can see that worked 😸